license/mapper/finder.go

```
@@ -25,7 +24,8 @@ func (f *Finder) License(ctx context.Context, m module.Module) (*license.License
 	// Look up the license by SPDX ID
 	lic, err := spdx.License(v)
 	if err != nil {
-		return nil, fmt.Errorf("Override license %q SPDX lookup error: %s", v, err)
+		// allow to use custom licence without SPDX ID defined
```

now the error is swallowed, maybe propage somehow or log with debug?